src/style.css

```diff
@@ -1,3 +1,59 @@
 body {
   margin: 0;
+  background-color: rgb(154, 154, 154);
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+.header {
+  padding-left: 50px;
+  padding-right: 50px;
+}
+
+.nav {
+  display: flex;
+  background: #fff;
+  height: 60px;
+  justify-content: flex-end;
+  align-items: center;
+  font-size: 12px;
+  text-transform: uppercase;
+}
+
+.nav__link {
+  text-decoration: none;
+  padding: 20px;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+  padding: 22.5px;
+  padding-left: 0;
+  padding-right: 0;
```

Comment:
Be consistent with your *vertical* margins (Add only top or only bottom margin, but not both). Here you're adding padding to the top and bottom, which can lead to unexpected results due to margin collapse. Consider using only one direction (either top or bottom) for vertical spacing.